src/main/java/eu/dissco/core/translator/Profiles.java

```diff
@@ -1,7 +1,7 @@
 package eu.dissco.core.translator;
 
 public class Profiles {
-  public static final String BIO_CASE = "bioCase";
+  public static final String BIOCASE = "biocase";
```

Comment:
Officially it is called: BioCASe, but lowercase should be better for code :)

Reply:
Yeah that's why... did this wrong a couple of times and then thought, just keep it simple